scenario28/base-setup.sh

```diff
@@ -18,12 +18,12 @@
 #TODO:read below property from infra.json file
 appName="travelocity.com"
 tomcatHost=$tomcatHost
-tomcatPort=8080
+tomcatPort=$tomcatPort
 tomcatUsername=scriptuser
 tomcatPassword=scriptuser
 tomcatVersion=7
 serverHost=$serverHost
-serverPort=443
+serverPort=$serverPort
```

Comment:
for the moment serverPort is not providing as an input parameter from TG, so plz correct it as 443